src/devicemanager/devicemanager.cpp

```diff
@@ -147,6 +153,28 @@ DeviceManager* DeviceManager::self()
     return self;
 }
 
+void DeviceManager::doBroadcast()
+{
+    QByteArray datagram = "Discovery";
```

Comment:
Why are you sending a "Discovery" string ?

Reply:
This is as documented in the "Communications Protocol v1.7.pdf"